CHANGES/7078.feature

```diff
@@ -0,0 +1 @@
+Implement ``WebSocketResponse.get_extra_info`` method to avoid accessing the private member ``_writer``
```

Comment:
Change notes should normally use the past tense as their audience is the end-users seeking info on the differences from the previous version. This also means that they don't need to see any internal implementation details, unlike devs/contributors that would be using Git commits for context.
A trailing period wouldn't hurt either.

Reply:
fixed